onecodex/viz/__init__.py

```diff
@@ -120,7 +120,7 @@ def stderr_filter(line):
 
         altair_module.renderers.enable(
             "altair_saver",
-            fmts=["html", "svg"],
```

Comment:
can you double-check with @clausmith whether the SVG saver is needed for anything? Not sure if removing it will break anything downstream

Reply:
This change is related to `test_reports.py` beginning to fail in the middle of working on this branch. I suspect it's related to a dependency that changed. I spent a long time banging my head against the wall trying to fix it. This is what eventually worked, and I can't really explain why.
You're right that we need to check in with @clausmith about its use downstream. One question I have is how the `onecodex` library used in report generation. This is an integration testing issue; we should not assume we can rely on `test_reports.py` to cover all cases if the associated library code is used for production downstream.

Reply:
a little more info on this: https://github.com/onecodex/onecodex/pull/426#discussion_r911546693

Reply:
I added a comment to the thread on Gerrit's PR

Reply:
fixed upstream and rebased